examples/agones-xonotic-xds/proxy-pool.yaml

```diff
@@ -43,7 +43,7 @@ spec:
       containers:
         - name: quilkin
           image: us-docker.pkg.dev/quilkin/release/quilkin:0.4.0
-          args: ["run", "--management-server", "http://quilkin-manage-agones:80"]
+          args: ["proxy", "--management-server", "http://quilkin-manage-agones:80"]
```

Comment:
Can we take `us-docker.pkg.dev/quilkin/release/quilkin:0.4.0` ➡ `us-docker.pkg.dev/quilkin/release/quilkin:0.5.0`, since the new syntax won't work with 0.4.0?